README.md

```diff
@@ -1,180 +1,108 @@
-to improve npm iteratively, with thorough consideration of each step along
-the way.
+Initial RFC proposals should be reviewed by the npm team and assigned to the agenda of an upcoming RFC discussion within 2 weeks of being opened. It is encouraged for the author of an RFC to attend this call, but if they are not able to for any reason discussions during the call will be summarized in the issue and responses can be gathered asynchronously.
 
-### Implementation as Exploration
+All discussions surrounding an RFC are covered by the [npm Code of Conduct](https://www.npmjs.com/policies/conduct). Please keep conversations constructive, civil, and low-temperature. If tensions flare during discussion, the npm team may, at its own discretion, moderate, remove, or edit posts, as well as locking the discussion on that PR or the entire RFCs repository.
 
-Typically, RFCs are discussed and ratified prior to implementation.
-However, this is not always the case!  Occasionally, we will develop a
-feature then write an RFC after the fact to describe and discuss it prior
-to merging into the latest npm release.
+## How do RFCs get ratified?
 
-Very often, an RFC will be difficult to examine without running code.  In
-those cases, we may opt to develop a proof of concept (or even fully
-production-ready implementation) of an RFC in process, in order to test it
-in reality before accepting it.
+When the scheduled RFC call takes place, the npm team and the community will read and discuss the changes detailed in the RFC. It is possible that the discussion will be carried over to a future RFC call due to time constraints or needing further discussion. Once there is rough consensus amongst the participants of the call and all objections have been considered, an RFC is considered ratified and will be labeled as "ready for work".
 
-Even when an RFC _is_ accepted, during implementation it is common to note
-additional objections, features, or decisions that need to be made.  In
-these cases, we may propose an amendment to a previously ratified RFC.
+It is common for an RFC to require multiple rounds of editing to address concerns brought up in the discussion.
 
-### Final Authority
+The RFC may be rejected altogether at the discretion of npm collaborators.
 
-The ultimate authority as to the ratification of any given RFC proposal is
-the npm CLI team, as they have ultimate authority over the direction and
-development of the actual thing these RFCs are about.
+Reasons for this may include, but are not limited to:
 
-## How do I create an RFC?
+- Objections are raised that are deemed to be relevant to the npm CLI, and cannot be reasonably addressed within the RFC.
+- The feature conflicts with another intended feature, or otherwise does not align with the future development direction of the npm CLI.
+- The feature described is prohibitively difficult to implement.
+- The feature described is better addressed by an alternate proposal.
```

Comment:
might be good to call out that it may also be objected for scope that goes beyond the CLI